internal/rest/dequeue_nodes.go

```diff
@@ -979,6 +979,16 @@ func (rest *RestOperations) RefreshCacheForRetryLayer(parentVsKey string, aviObj
 					poolObjName = *rest_op.Obj.(avimodels.Pool).Name
 				}
 				aviObjCache.AviPopulateOnePoolCache(c, utils.CloudName, poolObjName)
+				vsObjMeta, ok := rest.cache.VsCacheMeta.AviCacheGet(aviObjKey)
```

Comment:
@swathinsankaran: 409 indicated confilct in cache and just cache refresh should resolve issue of cache confilct.  Why do we require to update status over here? I think problem might be somewhere else.

Reply:
I agree. This is not required as the retry logic should solve it for the pool when a 409 is encountered.